docs/docs/api/optimizers/GEPA.md

```diff
@@ -68,6 +68,300 @@ highest_score_achieved_per_task = new_prog.detailed_results.highest_score_achiev
 best_outputs = new_prog.detailed_results.best_outputs_valset
 ```
 
+## Custom Instruction Proposers
+
+### What is instruction_proposer?
+
+The `instruction_proposer` is the component responsible for invoking the `reflection_lm` and proposing new prompts during GEPA optimization. When GEPA identifies underperforming components in your DSPy program, the instruction proposer analyzes execution traces, feedback, and failures to generate improved instructions tailored to the observed issues.
+
+### Default Implementation
+
+By default, GEPA uses the built-in instruction proposer from the [GEPA library](https://github.com/gepa-ai/gepa), which implements the [`ProposalFn`](https://github.com/gepa-ai/gepa/blob/main/src/gepa/core/adapter.py). This default proposer:
+
+- Uses a comprehensive prompt template that analyzes task context, inputs, outputs, and feedback
```

Comment:
What are the tradeoffs with having this list vs actually reproducing the default prompt directly? I feel like showing the default prompt could enable the end-user to judge whether they want to use it or not. Writing it this way makes it seem like there is no need to use custom instruction proposal!